src/Components/EnvironmentPicker/EnvironmentPicker.tsx

```diff
@@ -1,585 +1,518 @@
 import {
-    ComboBox,
     DefaultButton,
     Dialog,
     DialogFooter,
     DialogType,
     FontIcon,
     FontSizes,
-    IComboBoxOption,
-    IComboBoxStyles,
-    Icon,
     IconButton,
     IDialogContentProps,
```

Comment:
I'm thinking we should put the storage container name as the header for the list of containers instead of subscription.
![image](https://user-images.githubusercontent.com/57726991/184447581-e71c33e5-559d-46fe-81fb-370e76589454.png)

Reply:
do you mean storage account name? because these are already the container names. If so, I still dont want to change it for the sake of consistency with other resource pickers in the UI which is showing resources divided by subscription name. Because if we use subscription name header for this resource picker but subscription name for other resource pickers would be confusing.

Reply:
I actually disagree that it's confusing to change it. The pickers are a hierarchy. The first is a list of storage accounts within different subscriptions. The second is a list of contains within the selected storage account. There will NEVER be a container from another subscription so it makes no sense to call out the subscription that container is in. It would make more sense to either have _no_ header OR to have the storage account that was selected in order to convey that we are filtering the containers based on the previous selection.

Reply:
can't validate this one since I can't get it to fetch any containers